README.md

```diff
@@ -29,9 +27,13 @@ For platform builders, KubeVela is the core engine that empowers them to create
 
 ![alt](docs/resources/how-it-works.png)
 
-## Quick Start
+## Core Concepts
+
+Learn KubeVela starting from its [core concepts](https://kubevela.io/#/en/concepts).
+
+## Installation
 
-Quick start guides are available on [this section](https://kubevela.io/#/en/quick-start).
```

Comment:
the original link is right, why change quick start link to installation?

Reply:
@wonderflow Yes, and we need to add a pure k8s quick start doc. Actually I have already created a issue for you:  https://github.com/oam-dev/kubevela/issues/1054